CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs need to be appropriately assigned and labelled.
+
+1. Fork this repository.
+2. Clone your fork of the repository.
+3. When developing, develop and commit in a new, local branch. Leave the protected branches (e.g. develop) alone. They are needed to fetch from upstream.
+   1. If your changes are comprised of multiple commits, use `git rebase -i` to reduce the changes to a single commit. Otherwise, use `git amend` to add files to a commit.
```

Comment:
I'd say that changes should be atomic (or as atomic as possible). That being said, sometimes it's fine to put multiple commits into a pull request, if they are meaningful (see: #218).

And by atomic changes, I mean something like this:
<img width="958" alt="image" src="https://user-images.githubusercontent.com/4959931/171621342-d01cc879-1f41-47ea-af7b-31f20ac6e764.png">

I could merge it with other changes in the same pull request, but then it would be impossible to revert it or lookup the history for that fix.

Reply:
Changed to clarify that changes should be atomic.